src/components/icons/Alert/Alert.tsx

```diff
@@ -19,4 +20,15 @@ const AlertIcon: React.FunctionComponent<IAlertProps> = ({ fillColor = colors.ba
   </svg>
 );
 
-export default AlertIcon;
+/**
+ * @deprecated `<AlertIcon />` is deprecated and will be removed in: `4.0.0`. Use [this font-awesome 'exclamation-triangle' icon](https://fontawesome.com/icons/exclamation-triangle?style=solid) instead.
+ */
+export default deprecate(
+  AlertIcon,
+  `
+  ❗️ [@zopauk/react-components]
+  
+  <AlertIcon /> is deprecated and it will be removed in the next major version: 4.0.0. 
```

Comment:
but we can't use `<i class="fas fa-exclamation-triangle" />` in the actual version. In my opinion, it bill be better to remove the `use: <i class="fas fa-exclamation-triangle" /> instead` part.

Reply:
You can if you import **font-awesome** in the client app. We don't plan to bundle it in this library and we'll suggest them to use [`react-fontawesome`](https://github.com/FortAwesome/react-fontawesome) instead.